DESCRIPTION

```diff
@@ -1,9 +1,11 @@
 Package: dfeshiny
 Title: DfE R-Shiny Standards
 Version: 0.1.1
-Authors@R: 
+Authors@R: c(
     person("Rich", "Bielby", , "richard.bielby@education.gov.uk", role = c("aut", "cre"),
-           comment = c(ORCID = "0000-0001-9070-9969"))
+           comment = c(ORCID = "0000-0001-9070-9969")),
+    person("Charlotte", "Foster", , "charlotte.foster@education.gov.uk", role = c("aut"))
+)
```

Comment:
This bracket doesn't breaks everything if it's on a new line. If you put it at the end of line 7 instead, then lintr should work and you should then be able to run devtools::document().